app/app.go

```diff
@@ -325,6 +325,15 @@ func NewStargazeApp(
 	app.upgradeKeeper = upgradekeeper.NewKeeper(skipUpgradeHeights, keys[upgradetypes.StoreKey], appCodec, homePath)
 	app.registerUpgradeHandlers()
 
+	app.claimKeeper = claimkeeper.NewKeeper(
+		appCodec,
+		keys[claimtypes.StoreKey],
+		app.accountKeeper,
+		app.bankKeeper,
+		stakingKeeper,
+		app.distrKeeper,
+	)
```

Comment:
@shanev @JakeHartnell not really sure if we should be passing the reference `&stakingKeeper`

Reply:
Yeah, maybe it's not needed? Unless we want them to do a claim action for staking their $STARS...

Reply:
Yeah I kept it in case there's a claim to stake/delegate.